### PR TITLE
fixed: Swage theme - Config dropdown menu on small resulutions

### LINKED ISSUE #4511: [BUG] Theme: Swage - config dropdown not fully reachable on smaller screens
**Describe the bug**
see description and discussion on https://github.com/FreshRSS/FreshRSS/pull/4493#issuecomment-1219253651

**To Reproduce**
1. have a smaller screen (width > than mobile view, height < than config dropdown height)
2. open config menu
3. cannot reach the last items of the dropdown height

**Expected behavior**
All items in the list are reachable

**Environment information (please complete the following information):**
- FreshRSS version: 1.19.2, 1.20.0-dev

**Additional context**
While 150% zoom is recommended in Windows on some (or all?) screens, we have to keep an eye on it.

Right now only Swage theme is affected, because the header is fixed. Other themes can be scrolled down.

### PR DESCRIPTION
**Pull request by math-GH**
Closes #4511

After:
Dropdown menu is scrollable
grafik

Changes proposed in this pull request:

- (S)CSS

How to test the feature manually:

1. see #4511

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
@Alwaysin please have a look here :) The fix was easier than I thought before.

File: p/themes/Swage/swage.scss [hunk 1]
```diff
@@ -838,8 +838,13 @@ form {
 			margin-top: 3px;
 		}
 
-		.dropdown .icon {
-			filter: brightness(3);
+		.dropdown .dropdown-menu {
+			max-height: calc(100vh - 45px);
+			overflow: scroll;
+
+			.icon {
```
Comment (Frenzie) [inline p/themes/Swage/swage.scss:845]: Is this change on purpose? It looks quite unrelated to the overflow stuff.

Comment (math-GH) [reply p/themes/Swage/swage.scss:845]: it was by purpose.

`.dropdown .icon` is the same like `.dropdown .dropdown-menu .icon`, so `.dropdown .dropdown-menu` can be used for the (new) scrolling and for the `.icon`

**Comment by Alwaysin** [comment]
> @Alwaysin please have a look here :) The fix was easier than I thought before.

I tried it and it works as expected!
